mkdocs_monorepo_plugin/parser.py

```diff
@@ -56,7 +56,7 @@ def removeMkdocsYmlFromPath(path):
             rootDir = os.path.normpath(os.path.join(
                 os.getcwd(), self.config['config_file_path'], '../'))
             absPath = os.path.normpath(os.path.join(
-                rootDir, re.sub(r'\/mkdocs.yml$', '', path, re.IGNORECASE)))
+                rootDir, re.sub(r'(.*)\/.*\.yml$', r'\1', path, re.IGNORECASE)))
```

Comment:
Wondering if you could add a comment here explaining the regex. :)

Reply:
Sure, added a commit.